serverless/networking/serverless-ossm-tls.adoc

```diff
@@ -0,0 +1,37 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-tls"]
+= Configuring TLS for a custom domain using {ProductName}
+:context: serverless-ossm-tls
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+You can create a TLS key and certificates for a custom domain and subdomain using {ProductName}.
+
+[NOTE]
+====
+{ServerlessProductName} is compatible only with full implementations of either {ProductName} 1.x or 2.x. {ServerlessProductName} does not support custom usage of some 1.x resources and some 2.x resources in the same deployment. For example, upgrading to 2.x while still using the control plane `maistra.io/v1` spec is not supported.
+====
+
+[id="serverless-ossm-tls-prereqs"]
+== Prerequisites
+
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install {ProductName} xref:../../service_mesh/v1x/installing-ossm.adoc#installing-ossm-v1x[1.x] or xref:../../service_mesh/v2x/installing-ossm.adoc#installing-ossm-v2x[2.x].
+* Complete the configuration steps in xref:../../serverless/networking/serverless-ossm.adoc#serverless-ossm[Using Service Mesh with OpenShift Serverless].
+* Configure a custom domain. See xref:../../serverless/networking/serverless-ossm-custom-domains.adoc#serverless-ossm-custom-domains[Using custom domains for Knative services with Service Mesh].
+* In this example, `openssl` is used to generate certificates, but you can use any certificate generation tool to create these.
+
+[IMPORTANT]
+====
+This example uses the `example.com` domain.
+The example certificate for this domain is used as a certificate authority (CA) that signs the sub-domain certificate.
```

Comment:
This PR uses both "subdomain" and "sub-domain". Pretty sure it should be no hyphen - there are two instances in this file that need updated.